R/modify_legend.R

```diff
@@ -149,30 +149,36 @@ combine_legend_args <- function(object, new.legend.args, ..., where){
   
   legend.args <- object[['legend']][['legend.auto']]
   
-  orderedParams <- new.legend.args[match(names(legend.args), names(new.legend.args))]
-  for (j in seq_along(legend.args)) {
-    if (where == 'first'){
-      legend.args[[j]] <- c(orderedParams[[j]], legend.args[[j]])  
-    } else {
-      legend.args[[j]] <- c(legend.args[[j]], orderedParams[[j]])  
-    }
+  which.overall.args <- which(names(legend.args) %in% get_legend_arg_names(overall = TRUE))
```

Comment:
kind of prefer returning a logical vector here (instead of `which`) and then using `[!overall.args]` below